lib/bios/menus.robot

```diff
@@ -507,3 +507,40 @@ Boot System Or From Connected Disk
         ${system_index}=    Get Index Of Matching Option In Menu    ${menu_construction}    ${system_name}
     END
     Press Key N Times And Enter    ${system_index}    ${ARROW_DOWN}
+
+Check Platform Sleep Type Can Be Selected
```

Comment:
menus.robot is getting long already.

If we need specific keywords for specific menus, we may need another file, as we have for secure boot, for instance.

Reply:
moved to `/lib/sleep-lib.robot` in e8394a4e17ed0293f131ab9bf39410ab7df5ae0f
Kwd name changed to `Check If Platform Sleep Type Can Be Selected`  https://github.com/Dasharo/open-source-firmware-validation/pull/128/files#diff-0d4b3b4e8afbeb5a9e8e3655f8d6ecf853f6e7e246a3da142cca93bd987e402eR6
Documentation added.